server/server.go

```diff
@@ -1246,6 +1249,15 @@ func (s *Server) v1QueryGet(w http.ResponseWriter, r *http.Request) {
 	}
 	qStr := qStrs[len(qStrs)-1]
 
+	unsafeBuiltins, err := validateQuery(qStr)
+	if err != nil {
+		writer.ErrorAuto(w, err)
+		return
+	} else if len(unsafeBuiltins) > 0 {
+		writer.Error(w, http.StatusBadRequest, types.NewErrorV1(types.CodeInvalidParameter, "unsafe builtins in query: %v", strings.Join(unsafeBuiltins, ",")))
```

Comment:
nit: `s/builtins/built-in function calls/`

Reply:
Done.